libvirt/tests/src/migration/migration_with_disk/migration_with_vhostuser.py

```
@@ -0,0 +1,156 @@
+        test.log.info("Cleanup completed successfully.")
+
+    libvirt_version.is_libvirt_feature_supported(params)
+    vm_name = params.get("migrate_main_vm")
+    vm = env.get_vm(vm_name)
+
+    # Initialize variables
+    image_path = None
+    sock_path = None
+
+    # Back up xml file.
+    vmxml_backup = vm_xml.VMXML.new_from_inactive_dumpxml(vm_name)
```

Ruff (0.14.10)

Local variable `vmxml_backup` is assigned to but never used

Remove assignment to unused variable `vmxml_backup`

(F841)